src/mlagility/analysis/status.py

```diff
@@ -133,6 +142,8 @@ def print_invocation(
 
     print(f"{ident}\tInput Shape:\t{input_shape}")
     print(f"{ident}\tHash:\t\t" + invocation_hash)
+    print(f"{ident}\tBuild Name:\t" + build_name)
+    print(f"{ident}\tCache dir:\t" + cache_dir + '/' + build_name)
```

Comment:
Discussion (I dont have a super strong feeling either way):

Is it redundant to print both the build name and the full cache path? 
  - I can see it being convenient since `benchit cache stats BUILD_NAME` is a thing, and finding the whole dir tree on disk is also often useful. But on the 
  - The user has a few ways to get all the info without printing both:
    - If they only have the full cache path, the last last dir name in the path is the build name
    - If they just have the build name, `benchit cache location` will tell them the cache path

Reply:
Continued discussion [here](https://github.com/groq/mlagility/pull/333#discussion_r1240007644)